File: 3859-maximum-product-of-two-digits/maximum-product-of-two-digits.py
class Solution(object):
    def maxProduct(self, n):
        first, second = -1, -1
        while n>0:
            d=n%10
            if d>first:
                second=first
                first=d
            elif d>second:
                second=d
            n//=10
        return first * second
        

File: 3859-maximum-product-of-two-digits/maximum-product-of-two-digits.cpp
class Solution {
public:
    int maxProduct(int n) {
        vector<int> in;
        while(n>0){
            int ones=n%10;
            in.push_back(ones);
            n=n/10;
        }
        sort(in.begin(),in.end());
        int m=in.size();
        return in[m-1]*in[m-2];
    }
};